pinot-core/src/main/java/org/apache/pinot/core/transport/ServerChannels.java

```diff
@@ -147,6 +148,9 @@ synchronized void sendRequest(InstanceRequest instanceRequest)
       _channel.writeAndFlush(Unpooled.wrappedBuffer(requestBytes), _channel.voidPromise());
       _brokerMetrics.addMeteredGlobalValue(BrokerMeter.NETTY_CONNECTION_REQUESTS_SENT, 1);
       _brokerMetrics.addMeteredGlobalValue(BrokerMeter.NETTY_CONNECTION_BYTES_SENT, requestBytes.length);
+      String tableName = instanceRequest.getQuery().getQuerySource().getTableName();
+      String rawTableName = TableNameBuilder.extractRawTableName(tableName);
+      _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.REQUEST_SIZE, requestBytes.length);
```

Comment:
This is not the user's request. Can you please change to get the query string length to be the request size?

Reply:
I am not sure if this will work:
```
instanceRequest.getQuery().toString().length());
```
where `instanceRequest.getQuery()` return a BrokerRequest which is a thrift object, I am not sure if `toString()` will return the original user query. 

If we can not get the user's original query at netty layer (query has been converted as thrift object), the only option is add the metric at SingleConnectionBrokerRequestHandler.java L93-94